supple (#1)
chancefocus/finance_benchmark: README.md

#### 2.2.4 FinNL
- Sourse：https://github.com/ssymmetry/BBT-FinCUGE-Applications/tree/main/FinCUGE_Publish
- Introduction： a financial news classification dataset.Given financial news articles, the model needs to classify them into up to 15 possible categories, with evaluation measured by F1-Score. The training set contains 8,000 articles, the validation set contains 1,000 articles, and the test set contains 1,000 articles.

#### 2.2.6 FinNA

> - Sourse：https://github.com/ssymmetry/BBT-FinCUGE-Applications/tree/main/FinCUGE_Publish

Reviewer: ```suggestion
- Source: paper? challenge?
```

> - Introduction：a financial news summarization dataset. Given financial news articles, the model needs to generate a summary, with evaluation measured by Rouge (Lin, 2004). The training set contains 24,000 articles, the validation set contains 3,000 articles, and the test set contains 3,000 articles.

Reviewer: ```suggestion
- Introduction：a financial news summarization dataset. Given financial news articles, the model needs to generate a summary, with evaluation measured by Rouge (Lin, 2004). The training set contains 24,000 articles, the validation set contains 3,000 articles, and the test set contains 3,000 articles.
```

#### 2.2.7 FinRE
- Sourse：https://github.com/ssymmetry/BBT-FinCUGE-Applications/tree/main/FinCUGE_Publish

> - Introduction：a financial news relation extraction dataset. Given financial news articles and head-tail entity pairs, the model needs to classify the relation between entity pairs into up to 44 categories, including the null relation, with evaluation measured by F1-Score. The training set contains 7,454 articles, the validation set contains 1,489 articles, and the test set contains 3,727 articles

Reviewer: ```suggestion
- Introduction：a financial news relation extraction dataset. Given financial news articles and head-tail entity pairs, the model needs to classify the relation between entity pairs into up to 44 categories, including the null relation, with evaluation measured by F1-Score. The training set contains 7,454 articles, the validation set contains 1,489 articles, and the test set contains 3,727 articles
- [Official address](https://github.com/ssymmetry/BBT-FinCUGE-Applications/tree/main/FinCUGE_Publish)
```

### 2.2 Chinese Datasets

#### 2.2.1 BQ Corpus（4.2M）

- Sourse：Chen J, Chen Q, Liu X, et al. The bq corpus: A large-scale domain-specific Chinese corpus for sentence semantic equivalence identification[C]//Proceedings of the 2018 conference on empirical methods in natural language processing. 2018: 4946-4951.
- Introduction： The issue-matching data for the banking and finance sector, which includes issue pairs drawn from a year's worth of online banking system logs, is the largest issue-matching data available for the banking sector.
- [Official address](https://www.luge.ai/#/luge/dataDetail?id=15)

> #### 2.2.2 CCKS2022金融领域FEW-SHOT事件抽取数据集

Reviewer: Since this is a readme in english, please translate the title in english

#### 2.2.1 BQ Corpus（4.2M）

> - Sourse：Chen J, Chen Q, Liu X, et al. The bq corpus: A large-scale domain-specific Chinese corpus for sentence semantic equivalence identification[C]//Proceedings of the 2018 conference on empirical methods in natural language processing. 2018: 4946-4951.

Reviewer: ```suggestion
- Source：Chen J, Chen Q, Liu X, et al. The bq corpus: A large-scale domain-specific Chinese corpus for sentence semantic equivalence identification[C]//Proceedings of the 2018 conference on empirical methods in natural language processing. 2018: 4946-4951.
```
